nixos/modules/system/boot/stage-2-init.sh

```diff
@@ -1,9 +1,15 @@
 #! @shell@
 
 systemConfig=@systemConfig@
+verbose="@verbose@"
```

Comment:
I think there still needs to be a boot.verbose flag added to stage-2.nix, similar to how it's done in stage-1.nix. Maybe it's a good idea to re-use the value of boot.initrd.verbose? I left this change out to get some feedback!